### PR TITLE
Get secrets from k8s and populate in github

### PR DESCRIPTION
**Pull request by aelkiss**
* Merge creation of environment secret vs. repo secret into single class
* Provide info on command line.
* Extract secret creation to separate class

No tests; not sure how valuable they are without actually interacting with GitHub. Could use webmock if we really want tests.

Next step is automatically extracting the info we need via `kubectl` and scripting creation of the secrets we actually need.

### CODE REVIEW

**Comment by aelkiss** [comment]
Tested this against https://github.com/hathitrust/oai_solr; was able to create GitHub secrets there.

That *doesn't* include `tanka` or `kubectl` in the docker image. Since the dependencies for the ruby script itself are relatively light I chose to just bundle install locally. Either way I should update the README with instructions/dependencies.

**Comment by aelkiss** [comment]
@niquerio  Still waiting for review.

With the latest commit here, I was able to populate secrets and use them to deploy with use the secrets to deploy -- https://github.com/hathitrust/datasets/runs/8214497533?check_suite_focus=true

File: set-k8s-secrets.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+if [[ ! -z "$environment" ]]; then
+  environment="-e $environment"
+fi
+
+set_secret () {
+  bundle exec ruby set_secret.rb $environment "$repository" "$@"
+}
+
+source $(dirname $(realpath $0))/.env
```
Comment (aelkiss) [inline set-k8s-secrets.sh:62]: Should print a helpful error if GITHUB_PAT isn't set or in `.env`?

**Comment by aelkiss** [comment]
Would also be good to have an option to run without tanka & just specify the kubectl context and namespace directly.

**Review by niquerio — APPROVED** [review]
I made some changes that I'd like you to look at. Namely the `README`, the usage message, and adding an error if there's not `GITHUB_PAT`.

**Review by aelkiss — COMMENTED** [review]
Updates to the README, etc. look good to me.

**Pull request merged**